index.html

```diff
@@ -1,13 +1,373 @@
 <!DOCTYPE html>
-<html lang="en">
-    <head>
-        <title>Design Teardown</title>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <link href="assets/style.css" rel="stylesheet">
-        <link href="assets/reset.css" rel="stylesheet">
-    </head>
-    <body>
-
-    </body>
+<html lang="zxx">
+
+<head>
+    <title>Design Teardown</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="assets/style.css" rel="stylesheet">
+    <link
+        href="https://fonts.googleapis.com/css?family=Roboto:100,100i,300,300i,500,500i,700,700i,900,900i&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <div class="navbar">
+            <div class="logo">
+                <a href="#">
+                    Smashing
+                    Magazine
+                </a>
+            </div>
+            <div class="nav">
+                <ul>
+                    <li>
+                        <a href="#">
+                            <img src="https://d33wubrfki0l68.cloudfront.net/2bc30327ebf529a249e825447d86f18546a1d6db/36b19/images/nav-icons/articles.svg"
+                                alt="" width="39" height="44">
+
+                            <h2>Articles</h2>
+                            <p>Lorem Ipsum </p>
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="https://d33wubrfki0l68.cloudfront.net/2bc30327ebf529a249e825447d86f18546a1d6db/36b19/images/nav-icons/articles.svg"
+                                alt="" width="39" height="44">
```

Comment:
remove the selectors width and height in line #37 as no inline styling should be in the HTML file